src/termData.js

```diff
@@ -0,0 +1,64 @@
+function Term(shortName, longName, startDate = []) {
+    this.shortName = shortName;
+    this.longName = longName;
+    this.startDate = startDate;
+    Object.seal(this);
```

Comment:
StackOverflow is pretty scarce on the topic but it seems like `Object.seal` (and `Object.freeze`, which _should_ have been the method used here…) is really only used to implement "proper" immutability as found in other programming languages. Since we won't be modifying these objects elsewhere, it's probably fine to remove it.